os/mm/mm_heap/mm_getheap.c

```diff
@@ -144,7 +145,7 @@ struct mm_heap_s *mm_get_app_heap_with_name(char *app_name)
 
 	/* Search the heap */
 	while (node && node->is_active) {
-		if (strncmp(node->app_name, app_name, BIN_NAME_MAX) == 0) {
+		if (strncmp(node->app_name, app_name, BIN_NAME_MAX + 1) == 0) {
```

Comment:
last is always null, BIN_NAME_MAX is correct, I think.

Reply:
Let's assume that `node->app_name` is "abc", `app_name` is "abcd" and BIN_NAME_MAX is 3.
Then this comparison returns OK, because it tries to check from index 0 to 2.
But it is not correct. 
So I think `BIN_NAME_MAX + 1` is correct.

Reply:
OK